### DISCUSSION TITLE
VIP JOSS paper

### DISCUSSION THREAD

**Discussion by VChristiaens** [vortex-exoplanet/VIP#525] [Announcements]
Dear VIP community,

A first draft of the [paper](https://github.com/vortex-exoplanet/VIP/blob/master/paper.pdf) to be submitted to the Journal of Open Source Software is now available in the GitHub repository of VIP. 

**Authorship** is open to all VIP contributors, whether you have (i) [contributed directly to the GitHub](https://github.com/vortex-exoplanet/VIP/graphs/contributors?from=2015-07-26&to=2022-04-27&type=a), (ii) provided code to either @carlos-gg or myself which was then included to VIP, (iii) supervised the project, or (iv) are currently developing new features for VIP as per the [development plan](https://github.com/vortex-exoplanet/VIP/projects/1). Authorship will be made on an opt-in basis -- mostly because I do not have the full name, institute, ORCID and email address for some of you. Feel free to forward this invitation to any contributor who you fear may not see the announcement.

If you are interested to be included as co-author in this publication, could you please do one of the following (ordered by preference):
1. sync the latest version of VIP on your fork; directly edit the paper.md file by adding your name, ORCID and institute; commit and push your changes; and submit a pull request; 
2. reply to this thread including your name, institute and ORCID as they should appear in the paper;
3. send an email to valentin.christiaens@uliege.be including your name, institute and ORCID as they should appear in the paper.

I would highly appreciate feedback on the draft. How? Depending on your choice above:
1. feel free to correct typos directly in the paper.md file, and leave comments in bold font (by surrounding text with double asterisks);
2. upload an annotated pdf of the draft, and share the link in your reply;
3. attach an annotated pdf of the draft to your email.

The paper should be at most 1000 words long, which is roughly the current length of the draft without references. Although the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#) suggest it is not a hard limit, I would prefer not to make it significantly longer. Please consider that when providing feedback.

Whether to provide your name/affiliation or feedback, please do so by **May the 4th**.

Thanks again for your contributions to VIP, and thanks in advance for your help with the draft!

Ps: I am currently finalising more automatic tests (to reach > 60% coverage) as well as the last jupyter notebook tutorial (on IFS data processing), which I expect both to be included to VIP by May the 4th. The expected VIP version to be associated to the paper is 1.3.0 - the upgrade will be made once the paper is accepted, in case additional inclusions are made (or some bugs fixed) by then.

**Comment by r4lv** [comment]
Dear Valentin,

thank you so much for your effort! I'm very happy to see how VIP evolved in the last years :)
The paper looks great! The only thing I noticed is a missing link to the main GitHub repo — or is that added automatically by JOSS?

**Reply by VChristiaens** [reply]
Hi Ralf,

Thanks for your reply - and thank you for all your contributions to the project (never got to say it in person)!

I had the same question regarding including the GitHub repo link or not. I checked a couple of published JOSS papers and couldn't find the GitHub repo link in the main body of those papers. I have noted that it is systematically made available on the JOSS page where the paper can be viewed and downloaded though.

Cheers,
Valentin
